cmd/root.go

```diff
@@ -267,7 +267,7 @@ func fetchGitRepositoryFromRubyGems(packageName string) (string, error) {
 	return v.SourceCodeURI, nil
 }
 
-func init() {
+func InitRootCmd() {
```

Comment:
Let's add `//nolint` here too and not change code logic.

Reply:
In that case, should we just remove this linter from .golangci.yml?

Reply:
Going forward we want to avoid using `init` and that is the reason to have this linter. So we should not be removing from the .golangci.yml

Reply:
Changed these to `//nolint`.

Reply:
Thank you!